### PR TITLE
Move to jest

### PR DESCRIPTION
**Pull request by evemontalvao**
**CHANGELOG** :memo:

* Remove Karma dependences, karma.conf.js and script from package.json
* Add getComputedStyle mock function
* Replace \_\_html\_\_ by fileSystem to import fixtures
* Add Jest scripts to package.json

### CODE REVIEW

File: spec/index.spec.js [hunk 1]
```diff
@@ -96,7 +122,16 @@ describe('MediaQueryContent', () => {
...
+      Object.defineProperty(window, 'getComputedStyle', {
+        value: () => ({
+            getPropertyValue: (prop) => {
+                return 'desktop';
+            }
+        })
+      });
+      document.body.innerHTML = fs.readFileSync(`${fixturesPath}/desktop.html`);
+      mediaQueryContent = new MediaQueryContent();
+
```
Comment (williamcosta) [inline spec/index.spec.js]: Blank line 💅

File: spec/index.spec.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
-import MediaQueryContent from '../src/dhalsim';
+import MediaQueryContent from '../src/dhalsim'
```
Comment (williamcosta) [inline spec/index.spec.js]: Missed the trailing comma 💅

File: spec/index.spec.js [hunk 3]
```diff
@@ -45,7 +47,15 @@ describe('MediaQueryContent', () => {
 
   describe('.breakpoint', () => {
     beforeEach(() => {
-      document.body.innerHTML = __html__['spec/fixtures/smartphone.html'];
+      Object.defineProperty(window, 'getComputedStyle', {
```
Comment (iondrimba) [inline spec/index.spec.js]: 👏 💯

File: spec/index.spec.js [hunk 4]
```diff
@@ -54,23 +65,25 @@ describe('MediaQueryContent', () => {
...
+      const { breakpoint } = mediaQueryContent;
 
       expect(breakpoint).toBe('smartphone');
     });
   });
 
   describe('.isMobile', () => {
     beforeEach(() => {
-      document.body.innerHTML = __html__['spec/fixtures/smartphone.html'];
+      getComputedStyleMock('smartphone');
```
Comment (iondrimba) [inline spec/index.spec.js:76]: Nice!!!

**Pull request merged**